Lib/test/test_importlib/test_util.py

```diff
@@ -757,5 +758,48 @@ def test_source_from_cache_path_like_arg(self):
  ) = util.test_both(PEP3147Tests, util=importlib_util)
 
 
+class MagicNumberTests(unittest.TestCase):
+    """
+    Test release compatibility issues relating to importlib
+    """
+    @unittest.skipUnless(
+        sys.version_info.releaselevel in ('final', 'release'),
+        'only applies to candidate or final python release levels'
+    )
+    def test_magic_number(self):
+        """
+        Each python minor release should generally have a MAGIC_NUMBER
```

Comment:
I think it would be better to write this as a comment rather than a docstring. If you want, add a one-line docstring, but most test methods don't have docstrings.